charts/ingress-nginx/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: ingress-nginx
-version: 2.13.0
+version: 2.13.1
```

As this is adding new functionality let's bump the minor version:
```suggestion
version: 2.14.0
```